tardis/transport/montecarlo/__init__.py

```diff
@@ -0,0 +1,30 @@
+"""
+Faciliating the MonteCarlo iterations. 
+
+During a simulation run, a number of MonteCarlo iterations specified
+in the configuration are run using the numba compiler.
+Most of the iterations are used to calculate the steady-state plasma 
+properties and with the last iteration, the spectrum is determined.
+"""
+
+"""
+Implements the main loop of the MonteCarlo routine.
+"""
+
+from llvmlite import binding
+
+binding.set_option("tmp", "-non-global-value-max-name-size=2048")
```

Comment:
this is something to bring up with the numba developers to see if this is important.